projects/igniteui-angular/src/lib/query-builder/query-builder-tree.component.html

```diff
@@ -7,7 +7,7 @@
 </ng-template>
 
 <ng-template #selectFromTemplate>
-    <div class="igx-filter-tree__inputs" [style.display]="(isInEditMode() && !this.isAdvancedFiltering()) ? 'flex' : 'none'">
+    <div class="igx-filter-tree__inputs" [style.display]="(isInEditMode() && (!this.isAdvancedFiltering() || this.isHierarchicalGridNestedQuery())) ? 'flex' : 'none'">
```

Comment:
I get the reason for the condition but could we name the function that switches it differently? Even if it's only ever going to be used for hierarchical grids, it will just add confusion to devs looking at it in the future as it implies some sort of hierarchical grid dependency. It could be `withoutFromSelectionMode` or something similar.

Reply:
The method is used in `getConditionList()` method and the suggested name will not make sense there.